documentation/modules/ROOT/pages/hcp-tech-breakdown.adoc

```diff
@@ -29,45 +31,53 @@ This operator is versioned in lockstep with a specific OCP version and is decoup
 The main responsibilities of this operator are:
 
 * Provisioning all the infrastructure required to host a control plane (it can be creating it or adopting existing one). The infrastructure can refer to management cluster resources, external cloud provider resources, etc.
-* Deploying an OCP control plane configured to run in the context of the provisioned infrastructure.
-* Implementing any versioned behavior necessary to rollout the new version of a control plane.
+(what a control plane needs as infrastructure is fixed, no? if yes can we list one by one?)
+* Deploying an OCP control plane configured to run in the context (what is context?namespaced?) of the provisioned infrastructure.
+* Implementing any versioned behavior necessary (any example) to rollout the new version of a control plane.
 
 [#hosted-cluster-config-operator]
 === Hosted Cluster Config Operator
-
+(maybe start with Operator is deployed by X and is responsible for, first sentence is confusing )
 Control plane component that is a peer to other control plane components (e.g: etcd, apiserver, controller-manager). It is managed by the Control Plane Operator in the same way as those other control plane components.
-
+(is there an operator per component? one for etc on for api server?)
 This operator is versioned in lockstep with a specific OCP version and is decoupled from the management cluster version.
 
 The main responsibilities of this operator are:
 
-* Reading CAs from the hosted cluster to configure the kube controller manager CA bundle running in the hosted control plane.
+* Reading CAs from the hosted cluster(it is not clear what is hosted cluster here and how the read happens?) to configure the kube controller manager CA bundle running in the hosted control plane.
```

Comment:
hosted cluster is the OCP cluster that has its control plane running as pods on a management cluster. The read happens thru the API.